spacemacs/packages.el

```diff
@@ -1677,14 +1677,19 @@ If ARG is non nil then `ag' and `pt' and ignored."
           (call-interactively (spacemacs//helm-projectile-do-search-find-tool
                                tools))))
 
+      (setq helm-projectile-sources-list '(helm-source-projectile-projects
+                                           helm-source-projectile-files-list
+                                           helm-source-recentf
+                                           helm-source-pp-bookmarks
+                                           helm-source-file-cache))
```

Comment:
That's the strength of Helm: we include many sources at once and don't care what in it. We simply type and narrow to the thing we need. That's why it was originally called `Anything`. We can remove project source. However, I don't think there's any trouble mixing project source and non-project source, since if you invoke the command outside of any project, you won't see it anyway. Fyi, this also solved the issue 5 [here](https://github.com/syl20bnr/spacemacs/issues/772), from the report of @chrisbarrett. Otherwise, we basically have no CtrlP if we remove everything as you said.